src/index.js

```diff
@@ -19,3 +21,4 @@ export {Tooltip} from "./Tooltip";
 export {Label} from "./Label";
 export {FileInput} from "./FileInput/FileInput";
 export * from "./InfoPanel";
+export {Number};
```

Comment:
haven't written a component before, why can't we do `export {Number} from "./Number"` like we do for the lines above?